src/OrleansServiceBus/Providers/Streams/EventHub/ICheckpointSettings.cs

```diff
@@ -0,0 +1,99 @@
+﻿
+using System;
+using System.Collections.Generic;
+using Orleans.Providers;
+
+namespace Orleans.ServiceBus.Providers.Streams.EventHub
```

Comment:
all instances of namespace Orleans.ServiceBus.Providers.Streams.EventHub  should be Orleans.ServiceBus.Providers